lib/Interpreter/IncrementalJIT.cpp

```diff
@@ -221,6 +222,11 @@ IncrementalJIT::getSymbolAddressWithoutMangling(llvm::StringRef Name,
     if (RuntimeDyld::SymbolInfo SymInfo = m_ExeMM->findSymbol(Name))
       return llvm::orc::JITSymbol(SymInfo.getAddress(),
                                   llvm::JITSymbolFlags::Exported);
+#ifdef LLVM_ON_WIN32
+    if (const void* Sym = platform::DLSym(Name))
```

Comment:
I now :-) agree that this is the right spot to add that windows specific code. I'd like to request two things:
- Could you add a FIXME that we need to get rid of the call to m_ExeMM->findSymbol (which in the end calls llvm::sys::DynamicLibrary::SearchForAddressOfSymbol), and replace it with our own
- In that "our own", we should revisit whether there is a way to create a hash map of all symbols, (or use / query the one from the Windows kernel for that process) instead of iterating through all opened libraries

One more note: this windows specific part should basically (or really?) replace the call to m_ExeMM->findSymbol(). But we have another call to that a few lines down. Should this get the same Windows-specific treatment?